@navikt/core/css/darkside/guide-panel.darkside.css

```diff
@@ -1,96 +1,124 @@
 .navds-guide-panel {
-  --__ac-guide-panel-guide-size: 4rem;
+  --__axc-guide-panel-guide-size: 4rem;
+  --__axc-guide-panel-guide-padding-top: calc((var(--__axc-guide-panel-guide-size) + 1rem) + var(--ax-spacing-05));
+  --__axc-guide-panel-arrow-width: 20px;
+  --__axc-guide-panel-arrow-height: 33px;
 
   position: relative;
-  padding-top: calc(var(--__ac-guide-panel-guide-size) / 2);
-}
-
-.navds-guide {
-  background: var(--ac-guide-panel-illustration-bg, var(--a-surface-alt-3-subtle));
-  border: 2px solid var(--ac-guide-panel-border, var(--a-border-alt-3));
-  border-radius: var(--a-border-radius-full);
-  overflow: hidden;
-  position: absolute;
-  width: var(--__ac-guide-panel-guide-size);
-  height: var(--__ac-guide-panel-guide-size);
-  left: 50%;
-  transform: translate(-50%, -50%);
-}
-
-.navds-guide svg,
-.navds-guide img {
-  height: 100%;
-  width: 100%;
-}
-
-.navds-guide-panel__content {
-  background-color: var(--ac-guide-panel-bg, var(--a-surface-default));
-  border: 2px solid var(--ac-guide-panel-border, var(--a-border-alt-3));
-  border-radius: var(--a-border-radius-large);
-  padding: var(--a-spacing-4);
-  padding-top: var(--a-spacing-12);
-}
-
-@media (min-width: 480px) {
-  .navds-guide-panel {
-    --__ac-guide-panel-guide-size: 6.25rem;
+  padding-top: var(--__axc-guide-panel-guide-padding-top);
+
+  .navds-guide {
+    background: var(--ax-bg-info-moderate);
+    border: 2px solid var(--ax-border-info);
+    border-radius: var(--ax-border-radius-full);
+    overflow: hidden;
+    position: absolute;
+    width: var(--__axc-guide-panel-guide-size);
+    height: var(--__axc-guide-panel-guide-size);
+    left: 50%;
+    transform: translate(-50%, calc(-1 * var(--__axc-guide-panel-guide-padding-top)));
```

Comment:
I'm not super duper happy about some of these bigger calculation blocks, but I couldn't immediately think of something smaller (they inevitably grow a bit with the use of our tokens 😅 )